packages/product-feed-zbozi/composer.json

```diff
@@ -32,21 +32,20 @@
         "shopsys/framework": "9.0.x-dev",
         "shopsys/migrations": "9.0.x-dev",
         "shopsys/plugin-interface": "9.0.x-dev",
-        "symfony/config": "^3.4",
-        "symfony/dependency-injection": "^3.4",
-        "symfony/form": "^3.4",
-        "symfony/http-kernel": "^3.4",
-        "symfony/translation": "^3.4"
+        "symfony/config": "^3.4|4.0",
```

Comment:
I do not think so, that we want to support older symfony anymore... We should restrict it to version 4.

Reply:
Hmm, it just made me sense for feeds packages :thinking:

Reply:
It is always possible to fix it in previous version of framework e.g. 7.3 :) this may lead to some unexpected behavior with Symfony 3.

Reply:
:+1: 3a7da1fae04303c8efed703da6694e4187dd2f5f